src/scoped_timer.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+class ScopedTimer
+{
+public:
+  ScopedTimer(const std::string& name) : name(name) {
+    t.start();
+  }
+  ~ScopedTimer() {
+    t.stop();
+    printf("[%s] time %f s\n", name.c_str(), t.time());
+  }
+private:
+  std::string name;
+  CGAL::Timer t;
```

Comment:
Is this really the timer you want?
https://doc.cgal.org/latest/Miscellany/classCGAL_1_1Timer.html

> When used to profile a multithreaded algorithm, this timer will add up the process time of each thread, which may be unexpected. To compute the wall-clock time of an algorithm, please consider using [CGAL::Real_timer](https://doc.cgal.org/latest/Miscellany/classCGAL_1_1Real__timer.html).

Reply:
Ooops indeed, fixed! I hope the different matters soon, maybe if/when corefinement becomes super parallel (@sloriot ;-))